src/core_modules/capture-core/components/WidgetErrorAndWarning/WidgetError/WidgetError.component.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React from 'react';
+import { colors } from '@dhis2/ui';
+import { Widget } from '../../Widget';
+import type { Props } from './WidgetError.types';
+import { WidgetErrorAndWarningContent } from '../content/WidgetErrorAndWarningContent';
+import { WidgetErrorHeader } from './WidgetErrorHeader';
+
+
+export const WidgetError = ({ error, classes }: Props) => {
+    const widgetType = 'error';
+
+    if (!error?.length) {
+        return null;
+    }
+
+    return (
+        <div
+            className={classes}
+            data-test={'error-widget'}
```

Comment:
There is no need to wrap the data test id with `{ }`. You can use directly the string id. Something like this 
`data-test="error-widget"`. There are other widgets in this PR where you can apply the same pattern.
Thank you!